### PR TITLE
Setup Commentstring & adds nvim includes to path

### PR DESCRIPTION
**Pull request by teto**

### CODE REVIEW

File: autoload/nvimdev.vim [hunk 1]
```diff
@@ -46,6 +46,11 @@ function! nvimdev#init(path) abort
   let c_makers = []
   let s:ctags_file = s:path . '/tmp/tags'
 
+  for inc in s:include_paths
+	echomsg inc
```
Comment (tweekmonster) [inline autoload/nvimdev.vim]: This needs to be removed.

File: autoload/nvimdev.vim [hunk 2]
```diff
@@ -46,6 +46,11 @@ function! nvimdev#init(path) abort
...
+    let &path.=','.inc
```
Comment (tweekmonster) [inline autoload/nvimdev.vim]: This should prefix `s:path/`.  I'm using absolute paths so things still work even if `:cd` is used later or auto-cd is disabled.

**Comment by teto** [comment]
done

**Comment by tweekmonster** [comment]
Thanks!

**Pull request merged**